repose-aggregator/components/services/datastore-service/datastore-service-impl/build.gradle

```diff
@@ -25,7 +25,11 @@ dependencies {
     testCompile "org.mockito:mockito-all"
     testCompile "org.codehaus.groovy:groovy-all"
     testCompile "org.hamcrest:hamcrest-all"
+    testCompile "org.springframework:spring-test"
+    testCompile "io.opentracing:opentracing-mock"
+    testCompile "org.apache.logging.log4j:log4j-core"
     testCompile "org.apache.logging.log4j:log4j-slf4j-impl"
+    testCompile "org.slf4j:jcl-over-slf4j"
```

Comment:
The two logging dependencies here look like they were added in anticipation of tests that verify log messages, but I don't see any such tests. For now, should we just drop these dependencies?

Reply:
The `org.apache.logging.log4j:log4j-slf4j-impl` was a premature add, but the `org.slf4j:jcl-over-slf4j` is needed because the `DistributedDatastoreServlet` immediately constructs a `Logger` and without the dependency throws a `ClassNotFoundException`.

The first one has been removed.